.github/workflows/phpunit-tests.yml

```diff
@@ -0,0 +1,37 @@
+name: Run PHPUnit Tests
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  phpunit:
+    name: Run tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php-version: ['8.2', '8.0', '7.4']
```

Comment:
I'd suggest 7.4 (the minimum) and 8.3 (latest), switching it to 8.4 as soon as it's out.  Not too many inbetween configurations we'd need to support

Reply:
Added in [05ccdde](https://github.com/aspirepress/AspireUpdate/pull/87/commits/05ccddea60af9002381837667339da0accdb88e2).

Reply:
Follow up: Limited to PHP 7.4 and 8.3 in [307d2f6](https://github.com/aspirepress/AspireUpdate/pull/87/commits/307d2f637e9021fd1be6a6caad2ee4b0419deaa6).